README.md

```diff
@@ -62,7 +62,7 @@ Here's an example:
 
```

Comment:
Please revert the README.md changes, it will be updated only on release (in order not to confuse the users with unreleased features)

Reply:
Done